Resources/public/css/views/dashboard-blocks.css

```diff
@@ -22,6 +22,13 @@
     margin-top: 1em;
 }
 
+.ez-view-dashboardblocksview .ez-view-dashboard-content-name {
```

Comment:
`.ez-view-dashboard-content-name` ? `ez-view-` prefix is reserved for view container class, I guess you meant `ez-dashboard-content-name`